src/Apps/Order/Components/CheckoutAddress.tsx

```diff
@@ -48,19 +72,126 @@ const VALIDATION_SCHEMA = Yup.object().shape({
+  const [modalTitle, setModalTitle] = useState<ErrorModalTitle>(
+    ErrorModalTitle.general
+  )
+  const [modalContent, setModalContent] = useState<ReactElement>()
+  const [selectedAddress, setSelectedAddress] = useState<
+    AddressSuggestionRadioButton
+  >(AddressSuggestionRadioButton.recommended)
+
+  const handleKeepAddressClick = () => {}
+
+  const handleEditAddressClick = () => {
+    setDisplayModal(false)
+  }
+
+  const handleUseSuggestedAddressClick = () => {}
+
+  const handleFormSubmit = async (address: Address) => {
+    const { firstLine, secondLine } = formatAddress(address)
+
+    const res = await mockRequest()
+    switch (res) {
+      case 0:
+        setModalTitle(ErrorModalTitle.general)
+        setModalContent(
+          <>
+            <Spacer y={2} />
+            <Text>
+              The address you entered may be incorrect or incomplete. Please
+              check it and make any changes necessary.
+            </Text>
+            <Spacer y={4} />
+            <Text fontWeight={600}>What you entered</Text>
+            <Spacer y={1} />
+            <Box border="1px solid #C2C2C2" p={20}>
+              <Text>{firstLine}</Text>
+              <Text>{secondLine}</Text>
+            </Box>
+            <Spacer y={4} />
+            <Flex width="100%" justifyContent="space-between">
+              <Button
+                onClick={handleKeepAddressClick}
+                variant="secondaryBlack"
+                width="49%"
+              >
+                Use This Address
+              </Button>
+              <Button onClick={handleEditAddressClick} width="49%">
+                Edit Address
+              </Button>
+            </Flex>
+          </>
+        )
+        setDisplayModal(true)
+        break
+      case 1:
+        setModalTitle(ErrorModalTitle.suggested)
+        setModalContent(
+          <>
+            <Spacer y={2} />
+            <Text>
+              To ensure prompt and accurate delivery, we suggest a modified
+              shipping address.
+            </Text>
+            <Spacer y={2} />
+            <RadioGroup
+              onSelect={selected =>
+                setSelectedAddress(selected as AddressSuggestionRadioButton)
+              }
+              defaultValue={selectedAddress}
+            >
+              <BorderedRadio
+                value={AddressSuggestionRadioButton.recommended}
+                label={AddressSuggestionRadioButton.recommended}
+                position="relative"
+              >
+                <Flex flexDirection="column">
+                  <Text variant="xs">{firstLine}</Text>
+                  <Text variant="xs">{secondLine}</Text>
+                </Flex>
+              </BorderedRadio>
+              <BorderedRadio
+                value={AddressSuggestionRadioButton.user_address}
+                label={AddressSuggestionRadioButton.user_address}
+                position="relative"
```

Comment:
Why relative?

Reply:
no need actually, I was trying to arrange the content alignment, using label solved my problem.